src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends Exception {
```

Comment:
Consider making the AuthenticationException a RuntimeException, so that it does not need to be explicitly caught or declared in method signatures.